### DISCUSSION TITLE
Where can I find the Overview-v2 dashboard

### DISCUSSION THREAD

**Discussion by choke55** [CarlosCuezva/dashboards-Grafana-Teslamate#26] [General]
Like the information in the dashboard Overview v2, but where can I pick up the j.son file?
Thanks 
Choke

**Comment by CarlosCuezva** [comment]
Here 

https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate/blob/main/dashboards/overview.json

**Reply by choke55** [reply]
Thank you very much - just download it and works perfect. Have a great Sunday and thanks for your quick answer.
Best regards
Choke
